_pages/how-we-work/tools/google-drive.md

```diff
@@ -2,32 +2,45 @@
 title: Google Drive
 ---
 
-Google Drive is a cloud storage and file backup system. We keep a lot of documentation on Google Drive, though we’re slowly switching material over to the Handbook.
-
+Google Drive is a cloud storage and file backup system. We keep a lot of documentation, project work, and templates in Google Drive.
 
 ## Setup
 
 Because Google Drive is a web application, you don't need to install anything.
 
 ## Usage
-We use Google Drive to keep track of documentation for different projects:
 
-* The [18F Team Folder](https://drive.google.com/a/gsa.gov/folderview?id=0B84F26FpUP0lR1B2VVNGSi1MMVk&usp=sharing_eid) contains all of our Google Drive. If you haven't yet, click the option to ["Add to My Drive"](https://support.google.com/drive/answer/2375057?hl=en) for this folder.
+We use Google Drive to keep track of our work and collaborate on documents, presentations, and spreadsheets.
 
+* **Abide by [the TTS Code of Conduct](/code-of-conduct).**  If you see anyone violating our Code of Conduct, see the reporting section.
+* When you’re at work, make sure you're on the 18F Google Drive and not your private account, especially when working with outside partners. Official 18F documents must live on the GSA/18F shared drive for compliance reasons.
 * Everything in Google Drive can be FOIA’ed.
+* Offline syncing for Google Drive/Docs has been disabled for GSA.
 
-* Blog drafts are written in Google Drive in the Blog folder. [Instructions are here](https://github.com/18F/blog-drafts/blob/master/README.md).
+### 18F Drive specifics
 
-* Every GSA employee can view and access docs. If another agency employs Google Docs, documents can be shared directly with them.  If the agency does not use Google Docs, [here are approved instructions for sharing documents with them](https://insite.gsa.gov/portal/content/517805).   Note that the agency staff could also create a google account using their work email [by using this form](https://accounts.google.com/SignUpWithoutGmail?hl=en).  Ask in [#teamops](https://gsa-tts.slack.com/messages/teamops) or [#it-issues](https://gsa-tts.slack.com/messages/it-issues) if you need help.
+* Visit [the 18F brand guide](https://brand.18f.gov/templates/) for default 18F document styles/templates.
 
-* Offline syncing for Google Drive/Docs has been disabled for GSA.
+* 18F blog drafts are written in Google Drive in the Blog folder. [Instructions are here](https://github.com/18F/blog-drafts/blob/master/README.md).
+
+* For 18F folks, the main way we collaborate on documents is by adding them to the [18F Team Folder](https://drive.google.com/drive/u/0/folders/0B84F26FpUP0lR1B2VVNGSi1MMVk) in a relevant sub-folder. (You can make a new sub-folder if it wouldn't fit into the existing folders.) This automatically gives the document the right permissions and makes it easy for the rest of the team to find.
```

Comment:
This is great and I actually have a Google Spreadsheet that I should move into the Team Folder ...